src/remote-config.ts

```diff
@@ -99,6 +100,7 @@ export const defaultRemoteConfig: RemoteConfig = {
   enable_non_transit_trip_search: true,
   enable_nynorsk: true,
   enable_on_behalf_of: false,
+  enable_onboarding_login: true,
```

Comment:
Could we get a race condition between Remote config and the onboarding? After app install the user needs Remote config to be loaded to get the `false` value, but RemoteConfig isn't part of things we wait for during app loading.

Reply:
Oh, then maybe 🤔 I'll try to test for it, but we _should_ probably wait for remote config on app launch then?

Reply:
Yeah, might be good to add that logic now. Note that we can't always wait for RemoteConfig on app start, since if the existing data is not outdated it won't be fetched.

Reply:
Added now. Done by setting a `isLoaded` flag in RemoteConfigContext after config has been fetched. Seems to works well when testing locally on fresh installs, and `isLoaded` becomes true right after default values are overwritten.